### ISSUE TITLE
JSON output required

### ISSUE THREAD

**Issue by indistinctTalk** [coinbase/assume-role#14]
Pretty unlikely that most people will be hitting this issue, but I needed to set the default output value in ~/.aws/config (example below) **before** opening a terminal to run assume-role or I would get error regarding invalid endpoint. Maybe there should be a safety check for this value to be present? 

Required minimum config:
```
[default]
output = json
```

Example of error if output value is missing:
```
> assume-role staging read
MFA Token:
Invalid endpoint: https://iam..amazonaws.com
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: argument --user-name: expected one argument
aws iam list-mfa-devices error
```

**Comment by grahamjenson** [comment]
Oh that is weird, I will leave this issue open until we update the readme.

**Comment by bboe** [comment]
I also encountered this error, however, the error response I get is:

```
parse error: Invalid numeric literal at line 2, column 0
parse error: Invalid numeric literal at line 2, column 0
parse error: Invalid numeric literal at line 2, column 0

An error occurred (InvalidClientTokenId) when calling the AssumeRole operation: The security token included in the request is invalid.
Failed to export session envars.
```

It seems like it would make sense to force the output to be json for the necessary requests, rather than rely upon what the user has things set to.

**Comment by grahamjenson** [comment]
Resolved with #40
